docs-zh/guide/getting-started.md

```diff
@@ -217,18 +217,18 @@ envd-quick-start        http://localhost:42779   envd-quick-start.envd   /home/g
 
 ## 更多文档 📝
 
-访问[envd 文档](https://envd.tensorchord.ai/guide/getting-started.html)。
+访问 [envd 文档](../guide/getting-started)。
 
```

Comment:
Linked to itself. Maybe we can delete this part.

Reply:
Done